seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/seatunnel/doris/util/JDBCUtils.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.doris.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class JDBCUtils {
+
+    public static Connection getConnection(String dorisbeaddress, String username, String password, String database) {
+        try {
+            Class.forName("com.mysql.jdbc.Driver");
+            String connectionURL = "jdbc:mysql://" + dorisbeaddress + "/" + database + "?useUnicode=true&characterEncoding=UTF8&useSSL=false";
+            return DriverManager.getConnection(connectionURL, username, password);
```

Comment:
Using jdbc connector connect to doris? you need add database dialect into jdbc-connector.

Or, reference doris native [client ](https://github.com/apache/doris-flink-connector/blob/master/flink-doris-connector/src/main/java/org/apache/doris/flink/backend/BackendClient.java) conncetion

Reply:
ok,Thank you very much for your advice. Because I don't know much about it, may I ask why we need to add the dialect of the database?

Reply:
[jdbc-connector ](https://github.com/apache/incubator-seatunnel/tree/dev/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/dialect)already supports the mysql driver

Reply:
Ok, I see. I will put it into the Connector-JDBC module